packages/xo-web/src/common/intl/messages.js

```diff
@@ -2271,6 +2275,8 @@ const messages = {
   proxyTestSuccessMessage: 'The proxy appears to work correctly',
   proxyLinkedRemotes: 'Click to see linked remotes',
   proxyLinkedBackups: 'Click to see linked backups',
+  proxyNetworkDnsPlaceHolder: 'Default to: 8.8.8.0',
+  proxyNetworkNetmaskPlaceHolder: 'Default to: 255.255.255.0',
```

Comment:
Default values are both in server and web, why not make it required server's side and only handle it in one place?

Reply:
Currently the web don't set default values, but it's a good idea to let only the web handle this case and set default values.

Reply:
We should avoid duplicating default values unless necessary, chose a side and stick to it.